.gitignore

```diff
@@ -76,3 +76,6 @@ hack/boilerplate/*.pyc
 
 # release notes
 _releasenotes
+
+# calico manifests archive
+release-*/manifests/calico-*.yaml
```

Comment:
is this needed?

Reply:
it's the temp location where the file is put when it's extracted, making sure it doesn't get checked in if the make target is interrupted before it gets moved for some reason

not 100% needed but more of a safety net, should I get rid of it?

Reply:
thanks, makes sense to keep